src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -8,16 +8,16 @@ namespace Microsoft.Framework.FileSystemGlobbing.Abstractions
 {
     public class DirectoryInfoWrapper : DirectoryInfoBase
     {
-        private DirectoryInfo DirectoryInfo;
+        private readonly DirectoryInfo _directoryInfo;
 
         public DirectoryInfoWrapper(DirectoryInfo directoryInfo)
         {
-            DirectoryInfo = directoryInfo;
+            _directoryInfo = directoryInfo;
         }
 
         public override IEnumerable<FileSystemInfoBase> EnumerateFileSystemInfos(string searchPattern, SearchOption searchOption)
         {
-            foreach (var fileSystemInfo in DirectoryInfo.EnumerateFileSystemInfos(searchPattern, searchOption))
+            foreach (var fileSystemInfo in _directoryInfo.EnumerateFileSystemInfos(searchPattern, searchOption))
```

Comment:
This is a general purpose library, not specifically used for Web. I don't think it should do any of those things by default.